jest.environment.js

```diff
@@ -0,0 +1,21 @@
+const JSDOMEnvironment = require('jest-environment-jsdom');
+const util = require('util');
+
+/**
+ * Custom Jest Environment based on JSDOMEnvironment to support TextEncoder and TextDecoder.
+ *
+ * ref: https://github.com/jsdom/jsdom/issues/2524
+ */
+class CustomJSDOMEnvironment extends JSDOMEnvironment {
```

Comment:
It looks like JSDOM does not support TextEncoder/TextDecoder. Therefore, we are using a custom Jest environment to run our tests in, which is based on JSDOM, but with support for TextEncoder/TextDecoder.